staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1613,6 +1613,11 @@ type VirtualMachineStartFailure struct {
 	RetryAfterTimestamp  *metav1.Time `json:"retryAfterTimestamp,omitempty"`
 }
 
+type VolumeMigration struct {
```

Comment:
Out of curiosity, do we anticipate this struct to have more fields in the future?

Trying to understand why the following will not work?
```
    VolumesForMigration []Volume
```

Reply:
We don't know. I have been talking with @mhenriks and wrapped in a struct exactly for this reason